ydb/tests/stress/reconfig_state_storage_workload/tests/test_scheme_board_workload.py

```diff
@@ -0,0 +1,9 @@
+# -*- coding: utf-8 -*-
+from ydb.tests.stress.reconfig_state_storage_workload.workload import WorkloadRunner
+from reconfig_state_storage_workload_test import ReconfigStateStorageWorkloadTest
+
+
+class TestReconfigStateStorageBoardWorkload(ReconfigStateStorageWorkloadTest):
+    def test_state_storage_board(self):
+        with WorkloadRunner(self.client, self.cluster, 'reconfig_state_storage_workload', 120, True, "SchemeBoard") as runner:
```

Comment:
А тут не должно быть:
```suggestion
        with WorkloadRunner(self.client, self.cluster, 'reconfig_scheme_board_workload', 120, True, "SchemeBoard") as runner:
```
?

То же самое в https://github.com/ydb-platform/ydb/blob/62ccf02ddd1557e4b249ebaa751832560503dd47/ydb/tests/stress/reconfig_state_storage_workload/tests/test_board_workload.py#L8

Если так сделано специально (переиспользуются пути), то ок, но надо добавить комментарий.

Reply:
Эти тесты изолированы друг от друга - но переименовать наверное стоит